src/hb-subset-plan.cc

```diff
@@ -389,8 +389,8 @@ _nameid_closure (hb_face_t *face,
  * Since: 1.7.5
  **/
 hb_subset_plan_t *
-hb_subset_plan_create (hb_face_t         *face,
-		       hb_subset_input_t *input)
+hb_subset_plan_create (hb_face_t	 *face,
```

Comment:
The function arguments need to be documented.

Reply:
Done.